src/Components/Header/Nav.jsx

```diff
@@ -0,0 +1,102 @@
+
+import "../../css/main.css";
+import "./Nav.css";
+
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import { Sidebar }  from "./sidebar";
+
+
+export const Nav = () => {
+  const [showSidebar, setShowSidebar] = useState(false);
+
+  const hamburgerHandler = () => {
+    setShowSidebar(!showSidebar);
+  }
+
+  const navLinks = [
+    { text: "Home", link:"/"},
+    { text: "Shop Now", link:"/products"}
+  ]
+  
+  return (
+    <div className="App">
+      <nav className="navbar flex pdr-4 pdl-3">
+
+            <div className="flex flex-align-center flex-gap-2">
+                <button className="btn-transparent display-small-screen" onClick={hamburgerHandler}>
+                  <span className="material-icons text-3xl">menu</span>
+                </button>
+
+                <h4 className="title font-bold m">Easy Mart</h4>
+
+                <ul className="list-style-none flex flex-gap-1">
+                  {
+                    navLinks.map((item) => {
+                      return(
+                        <li key={item.text}>
+                          <Link className="nav-links font-semibold ml-2 nav-home display-desktop" to={item.link}>{item.text}</Link>
```

Comment:
Here you can de-structure `item` & use `{link}` & `{name}` directly.

Reply:
yep, will do it. Thanks for reviewing.

>